praw/objects.py

```diff
@@ -1899,6 +1902,34 @@ def remove_editor(self, username, *args, **kwargs):
         return self.add_editor(username=username, _delete=True, *args,
                                **kwargs)
 
+    def revisions(self, **kwargs):
+        """Return a dictionary of all page revisions.
+
+        Additional parameters are passed to
+        :meth:`~praw.__init__.BaseReddit.request_json` to be used as
```

Comment:
IMO, kwargs should just be unpacked as usual, instead of being used specifically as params. This way, not only can the params keyword argument be used, but any others as well (if the user wants to use them).